lib/charms/mongodb/v0/helpers.py

```diff
@@ -82,16 +82,19 @@ def get_mongod_args(
     Returns:
         A string representing the command used to start MongoDB.
     """
-    #
     full_data_dir = f"{MONGODB_COMMON_DIR}{DATA_DIR}" if snap_install else DATA_DIR
     full_conf_dir = f"{MONGODB_SNAP_DATA_DIR}{CONF_DIR}" if snap_install else CONF_DIR
+    # in k8s the default logging options that are used for the vm charm are ignored and logs are
+    # the output of the container. To enable logging to a file it must be set explicitly
+    logging_options = "" if snap_install else f"--logpath={full_data_dir}/mongodb.log"
```

Comment:
just at small note: If we are using constants to build `full_data_dir` MONGODB_COMMON_DIR and CONF_DIR I think it make sense to use a constant for "mongodb.log", something like "MONGODB_LOG_FILE_NAME" or similar

Reply:
Good suggestion thank you @dmitry-ratushnyy